Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1487,7 +1494,7 @@ - (void)processNextQueueItem {
             }
             else if (![req isKindOfClass:[BranchOpenRequest class]] &&
                 (!self.preferenceHelper.deviceFingerprintID || !self.preferenceHelper.sessionID)) {
-                NSLog(@"[Branch Error] Missing session items!");
+                BNCLogError(@"Missing session items!");
```

Comment:
We can probably remove this log. Impossible case and it's not the developer's fault.